editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -340,6 +340,18 @@ void VisualShaderGraphPlugin::add_node(VisualShader::Type p_type, int p_id) {
 		Color(1.0, 1.0, 0.0), // sampler
 	};
 
+	static const Color vector_expanded_color[3] = {
+		Color(1.0, 0.0, 0.0), // red
+		Color(0.0, 1.0, 0.0), // green
+		Color(0.0, 0.0, 1.0) // blue
```

Comment:
I recall a recent pr making colours 30% more saturated. Not sure the details, but if it was merged, we can do it here too.

Reply:
Those colors are already fully saturated, but I recommend not using "pure" colors in UI design. Instead, you can reuse the colors used for axes in the 3D editor: https://github.com/godotengine/godot/blob/b6f381475fed44be21b8dd66cdb86e1a3b3af9cf/editor/editor_themes.cpp#L408-L410

(Don't hardcode those colors. Instead, get those colors using e.g. `get_theme_color("axis_y_color", "Editor")`.)

Reply:
Done